webapp/channels/src/components/app.tsx

```diff
@@ -16,20 +16,15 @@ const LazyRoot = React.lazy(() => import('components/root'));
 
 const Root = makeAsyncComponent('Root', LazyRoot);
 
-class App extends React.PureComponent {
-    render() {
-        return (
-            <Provider store={store}>
-                <CRTPostsChannelResetWatcher/>
-                <Router history={getHistory()}>
-                    <Route
-                        path='/'
-                        component={Root}
-                    />
-                </Router>
-            </Provider>
-        );
-    }
+function App() {
+    return (
+        <Provider store={store}>
+            <CRTPostsChannelResetWatcher />
+            <Router history={getHistory()}>
+                <Route path='/' component={Root} />
```

Comment:
nit: I think our linter is going to complain here. We do not allow multiple attributes in the same line.

Reply:
I could fix that if you want! Let me know :)

Reply:
It is going to be needed to pass the linter check anyway, so yes, please, do it. :slightly_smiling_face:

Reply:
i have done the requested changes, could you review it.